build-data-capturing-gradle-samples/capture-test-pts-support/gradle-test-pts-support.gradle.kts

```diff
@@ -86,3 +96,30 @@ class Capture(val logger: Logger) {
         }
     }
 }
+
+class KotestEngine(private val version: String) : Comparable<KotestEngine> {
```

Comment:
Can we use Gradle's `VersionNumber` class to do the version parsing/comparison here?

And maybe taking a step back, we want to compare `VersionNumber`s not `KotestEngine`s

Reply:
Updated to use `VersionNumber` to do the parsing/comparison.